srcpkgs/spotify-adblock/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'spotify-adblock'
+pkgname=spotify-adblock
+version=1.0.1
+revision=1
+build_style=gnu-makefile
+build_helper=rust
+conf_files="/etc/spotify-adblock/config.toml"
+hostmakedepends="make cargo"
+makedepends="rust-std"
+short_desc="Adblocker for Spotify"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/abba23/spotify-adblock"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=01edec4cc87fd0bbf77508fe6bcfdfba244771a992eca9d834df0f9399f1c360
+
+if [ "$CROSS_BUILD" ]; then
+	_releasedir="target/${RUST_TARGET}/release"
+else
+	_releasedir="target/release"
+fi
```

Comment:
unnecessary with cargo buildstyle

Reply:
oh i should expand on this, the cargo build style should put the .so file in `target/${RUST_TARGET}/release/` regardless of cross so it can just be put in the vinstall line below

Reply:
Are you sure?  
I just tried with cargo buildstyle, and this seems to be still needed...

Reply:
yes, I tried it myself. on x86_64 it gets put in `target/x86_64-unknown-linux-gnu/release` (or whatever the exact string is), I believe it's because of the `--target` [here](https://github.com/void-linux/void-packages/blob/master/common/build-style/cargo.sh#L8)

Reply:
> oh i should expand on this, the cargo build style should put the .so file in `target/${RUST_TARGET}/release/` regardless of cross so it can just be put in the vinstall line below

I hadn't seen this part before making my above comment :)

(Weird... github live-reloads the commits added to a PR, but doesn't load the comments.)